macros/src/codegen.rs

```diff
@@ -523,15 +518,15 @@ pub fn generate_code(sm: &ParsedStateMachine) -> proc_macro2::TokenStream {
 
     let guard_error = if sm.custom_guard_error {
         quote! {
-            /// The error type returned by guard functions.
-            type GuardError: core::fmt::Debug;
+            /// The error type returned by guard or action functions.
+            type ErrorType: core::fmt::Debug;
```

Comment:
Can we just make this `Error` to align with other crates?

Reply:
You mean rename GuardError to just Error ? I think it makes sense . Will do .